Main/Source/iconf.cpp

```diff
@@ -162,6 +164,10 @@ truthoption ivanconfig::AllWeightIsRelevant("AllWeightIsRelevant",
                                           "Only pile items with equal weight on lists", //clutter are useful now for crafting so their weight matters...
                                           "",
                                           false);
+truthoption ivanconfig::DropBeforeOffering("DropBeforeOffering",
+                                          "Drop the item on altar before offering it.",
```

Comment:
There shouldn't be a fullstop here, the short descriptions don't have them.

Reply:
"Drop the item on altar in case it is not accepted" :)